src/Roots/Acorn/Console/Commands/AcornInstallCommand.php

```diff
@@ -119,12 +126,18 @@ protected function askToStar(): void
             label: '🎉 All done! Would you like to show love by starring Acorn on GitHub?',
             default: true,
         )) {
-            match (PHP_OS_FAMILY) {
-                'Darwin' => exec('open https://github.com/roots/acorn'),
-                'Linux' => exec('xdg-open https://github.com/roots/acorn'),
-                'Windows' => exec('start https://github.com/roots/acorn'),
+            $opened = match (PHP_OS_FAMILY) {
+                'Darwin' => shell_exec('open ' . $this->repoUrl . ' 2>/dev/null') !== null,
+                'Linux' => shell_exec('xdg-open ' . $this->repoUrl . ' 2>/dev/null') !== null,
+                'Windows' => shell_exec('start ' . $this->repoUrl . ' 2>nul') !== null,
```

Comment:
I might be wrong about the behavior of `shell_exec()`, but afaik the first argument for start is the window title, i.e., `start <title> <command>`

```suggestion
                'Windows' => shell_exec('start "" ' . $this->repoUrl . ' 2>nul') !== null,
```
If you just run `start <url>`, it will open a new cmd prompt with the URL as the title of the prompt.

![start-url](https://github.com/user-attachments/assets/36369b40-9be2-499d-9f5b-dd6996fff256)

Changing it to `start "" <url>` opens the URL in my default browser.

Again, I didn't test with `shell_exec()`, though, since I don't have PHP installed in Windows.